docs/issues_catched.md

```diff
@@ -0,0 +1,34 @@
+
+# Issues Precaution finds
+
+Precaution utilizes multiple security static code analysis tools.
+That is the reason why the amount and types of checks for the different
+programming languages differs.
+The security static code analyzers are language-focused
+and some of the vulnerabilities they find are specific 
+for the programming language.
+However, some of those rules can be grouped:
+
+* Hardcoded credentials
+* SQL injections
+* Command injections
+* HTML injections
+* Weak random number generator
+* Imported and used weak cryptographic algorithms
+* Imported insecure libraries
+* Use of specific insecure functions from libraries
+* Weak key used for a cryptographic algorithm
```

Comment:
Move under "Weak cryptographic algorithms" (to group like items), generalise to "Weak cryptographic keys" or, to be inline with CWE, "Inadequate encryption strength"